src/ConnectorRuntime.ts

```diff
@@ -72,7 +77,9 @@ export class ConnectorRuntime extends Runtime<ConnectorRuntimeConfig> {
 
         const loggerFactory = new NodeLoggerFactory(connectorConfig.logging);
         ConnectorLoggerFactory.init(loggerFactory);
-        const runtime = new ConnectorRuntime(connectorConfig, loggerFactory);
+
+        const connectorMode = this.connectorModeFromConfig(connectorConfig);
+        const runtime = new ConnectorRuntime(connectorConfig, loggerFactory, connectorMode);
```

Comment:
Ich finds komisch, dass man den connectorMode extra reingeben muss, obwohl er eigentlich in der connectorConfig schon drin ist.